README.md

```diff
@@ -1 +1,230 @@
+    request, and leave a comment on the pull request explaining your changes.
+    Specifically, do not squash or rebase commits after they are visible on the
+    pull request.
+  - At some point, a maintainer will propose a "motion for final
+    comment period" (FCP), along with a *disposition* for the RFC (merge, close,
+    or postpone).
+    - This step is taken when enough of the tradeoffs have been discussed that
+    the maintainers are in a position to make a decision. That does not require
+    consensus amongst all participants in the RFC thread (which is usually
+    impossible). However, the argument supporting the disposition on the RFC
+    needs to have already been clearly articulated, and there should not be a
+    strong consensus *against* that position outside of the maintainers. 
+    Maintainers use their best judgment in taking this step, and the FCP itself
+    ensures there is ample time and notification for stakeholders to push back
+    if it is made prematurely.
+    - For RFCs with lengthy discussion, the motion to FCP is usually preceded by
+      a *summary comment* trying to lay out the current state of the discussion
+      and major trade-offs/points of disagreement.
+    - Before actually entering FCP, *all* maintainers must sign off;
+    this is often the point at which many maintainers first review the RFC
+    in full depth.
+  - The FCP lasts one week, or seven calendar days. It is also advertised
+    widely, e.g. in [Ursa Mailing
+    List](https://lists.hyperledger.org/mailman/listinfo/hyperledger-ursa). This
+    way all stakeholders have a chance to lodge any final objections before
+    a decision is reached.
+  - In most cases, the FCP period is quiet, and the RFC is either merged or
+    closed. However, sometimes substantial new arguments or ideas are raised,
+    the FCP is canceled, and the RFC goes back into development mode.
+
+## The RFC life-cycle
+[The RFC life-cycle]: #the-rfc-life-cycle
+
+Once an RFC becomes "active" then authors may implement it and submit the
+change as a pull request to the corresponding Ursa repo. Being "active" is not a rubber
+stamp, and in particular still does not mean the change will ultimately be
+merged; it does mean that in principle all the major stakeholders have agreed
+to the change and are amenable to merging it.
+
+Furthermore, the fact that a given RFC has been accepted and is "active"
+implies nothing about what priority is assigned to its implementation, nor does
+it imply anything about whether a Ursa developer has been assigned the task of
+implementing the feature. While it is not *necessary* that the author of the
+RFC also write the implementation, it is by far the most effective way to see
+an RFC through to completion: authors should not expect that other project
+developers will take on responsibility for implementing their accepted feature.
+
+Modifications to "active" RFCs can be done in follow-up pull requests. We
```

Comment:
Do follow-up PRs undergo the same procedure as a new RFC, or is it simpler to do a follow-up?

Reply:
It should be easier to touch up the language. If there's a substantial change to the design then it should be a new RFC. Maybe for small changes to previously approved RFCs we just revert to the existing PR approval, i.e. approval from two different maintainers who are different from the submitter and who ideally represent different viewpoints.